kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -1004,6 +1004,15 @@ object KyuubiConf {
       .stringConf
       .createOptional
 
+  val SESSION_CONF_FILE_RELOAD_DURATION: ConfigEntry[Long] =
+    buildConf("kyuubi.session.conf.file.reload.duration")
```

Comment:
cc @turboFei @ulysses-you in case you have other ideas on this configuration name

Reply:
sgtm